tests/shared-test-code/constants/zendeskParameters.ts

```diff
@@ -1,3 +1,14 @@
+export const CLOSE_ZENDESK_TICKET_COMMENT =
+  'Your ticket has been closed because some fields were invalid. Here is the list of what was wrong: From Date is in the future, To Date is in the future'
+
+export const ZENDESK_END_USER_NAME = 'Txma-team2-ticf-analyst-dev'
```

Comment:
I think we need to remove these? Might be they are still here due to all the conflicts. The idea now is that We have an object e.g. `zendeskConstants` which has a type where all the keys are readonly. This way you get auto-complete and things are a bit more organised.

Plus, we wanted to get rid of all the upper snake case variables knocking about. Although this is fine for environment vars since they are passed in from the shell (where this would be the convention in bash).